pytket/docs/build-docs.sh

```diff
@@ -8,9 +8,9 @@ PYTKET_VERSION="$(pip show pytket| grep Version | awk '{print $2}')"
 PACKAGE="pytket $PYTKET_VERSION"
 
 # Build the docs setting the html title to show the correct pytket version.
-sphinx-build -W -b html . build -D html_title="$PACKAGE API documentation"
```

Comment:
Did you intend to remove the `-W` flag here?

Reply:
Ah, forgot to put it back. It is helpful to remove when running locally so that it reports all warnings rather than just exiting after reaching the first one